wordpress-mobile/WordPress-Android
# Clean up NewSiteCreationBaseFormFragment

Fixes #8511 

I've removed few more things which we don't use and I missed them in the previous PR. Sorry 😞 

To test:
1. My Site
2. Switch Site
3. Plus Sign icon
4. Create WordPress site
5. Select a segment
6. Skip verticals
7. Skip site info
8. Select a domain
9. Set poor internet connection on emulator - or somehow make sure the create site requests fails
10. Click on the Create Site button
11. Wait until fullscreen error occurs
12. Click on the Contact Support button

Update release notes:

- [ x ] If there are user facing changes, I have added an item to `RELEASE-NOTES.txt`.
